app/javascript/entrypoints/application.js

```
@@ -0,0 +1,39 @@
+    const componentName = matches[1];
+    components[componentName] = module.default || module;
+  }
+});
+
+// Custom require context function for react_ujs
+const requireContext = (name) => {
+  return components[name];
+};
+
+// eslint-disable-next-line react-hooks/rules-of-hooks
+ReactRailsUJS.useContext({ require: requireContext });
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)